### PR TITLE
simple script to compare niimgs (e.g zmaps)

### PR DESCRIPTION
**Pull request by dohmatob**

### CODE REVIEW

File: qa/compare_niimgs.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        corr = stats.pearsonr(ref_img, src_img)[0]
+        corrs.append(corr)
+
+        if plot_hist:
+            ax1.scatter(ref_img, src_img, label="Pearsonr: %.2f" % corr, c="g",
+                        alpha=.6)
+            x = np.linspace(*ax1.get_xlim(), num=100)
+            ax1.plot(x, x, linestyle="--", c="k")
+            ax1.grid("on")
+            ax1.set_xlabel("img 1")
```
Comment (MartinPerez) [inline qa/compare_niimgs.py]: I think label names should be in the API with "imgs set 1", "imgs set 2" as the default value.

**Comment by MartinPerez** [comment]
Except for a small comment, LGTM. Thanks man! 

Could you please add this to utils.py, or maybe we should call this reporting.py and start filling any reporting related functions there, like this one? Please leave the file in the nistats folder.

**Comment by MartinPerez** [comment]
+ 1 MRG

**Comment by dohmatob** [comment]
Comments addressed.

File: nistats/reporting.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    log: boolearn, optional (default True)
+        Passed to plt.hist
+
+    plot_hist: boolean, optional (default True)
+        If True then then histograms of each img in ref_imgs will be plotted
+        along-side the histogram of the corresponding image in src_imgs
+
+    output_dir: string, optional (default None)
+        Directory where plotted figures will be stored.
+
```
Comment (bthirion) [inline nistats/reporting.py]: docstring missing for `ref_label` and `src_label`

**Comment by bthirion** [comment]
Thx. Please add a test.

**Comment by MartinPerez** [comment]
this was continued in #136

**Pull request closed without merging**